content/en/docs/concepts/sampling/index.md

```diff
@@ -103,7 +103,7 @@ as possible. A decision to sample or drop a span or trace is not made by
 inspecting the trace as a whole.
 
 For example, the most common form of head sampling is
-[Consistent Probability Sampling](/docs/specs/otel/trace/tracestate-probability-sampling-experimental/#consistent-probability-sampling).
+[Consistent Probability Sampling](https://github.com/open-telemetry/opentelemetry-specification/blob/main/oteps/trace/4673-experimental-probability-sampling.md).
```

Comment:
@carlosalberto @open-telemetry/specs-approvers - should links like these have been instead updated to refer to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/tracestate-probability-sampling.md?

For some context, see https://github.com/open-telemetry/opentelemetry-specification/issues/4687#issuecomment-3407105183.